lxc/file.go

```diff
@@ -11,38 +11,93 @@ import (
 	"strings"
 	"syscall"
 
+	"github.com/codegangsta/cli"
```

Comment:
Insert a new line after this import to keep it separate from the github.com/lxc/lxd/* ones.